README.md

```diff
@@ -51,5 +51,35 @@ This is an atlas scheme reduced to the information required by this exporter.
 - _Modules_ provide shared core functionality
     - _api_ &rarr; very basic functionality. Enables connectivity to ESXi-hosts etc. They are designed in an interchangeable manner. See helpers which are some kind of wrapper.
     - _helper_ &rarr; Are some kind of wrapper which combine different APIs in order to provide a unified interface. If an api-file gets interchanged, the helper/wrapper still expects the same format as before. For instance if you want to use a different SSH-library than paramiko then change the ssh-api-file in the api folder and make sure it still returns the console output of the remote host as before.
-    - _configuration_ &rarr; some small models that collect configuration from yaml, environment and commandline arguments in order to provide configuration in a dynamic and scoped manner to specific parts of the exporter.
-- exporter.py is the entry point. It dynamicly invokes collectors based on the configuration and sets up the whole program.
+    - _configuration_ &rarr; some small models that collect configuration from yaml, environment and command-line arguments in order to provide configuration in a dynamic and scoped manner to specific parts of the exporter.
+- exporter.py is the entry point. It dynamically invokes collectors based on the configuration and sets up the whole program.
+
+
+### Explainations
+
+
+**Landscape**
+![](assets/2021-03-30-10-31-58.png)
+
+1. Prometheus makes a HTTP-GET request on the exporter.
+2. The Exporter starts to collect information from ESXi-Hosts and the vCenter
+3. Finally a `Gauge-Metric` is build and will be sent back.
+
+
+**Structure of the project**
+![project structure](assets/2021-03-30-10-29-08.png)

```

Comment:
can we get some arrows and lines to see, how the stuff interacts with each other?

Reply:
yeah